build.gradle

```diff
@@ -194,6 +194,7 @@ dependencies {
   
   compile group: 'com.microsoft.azure', name: 'azure-servicebus', version: '1.2.8'
   compile group: 'io.vavr', name: 'vavr', version: '0.10.0'
+  compile group: 'org.json', name: 'json', version: '20180813'
```

Comment:
Could we double check it's not clashing with anything similar already present in the scope?

Reply:
I'm removing this dependency and leave the log message as it was before (using `EnvelopeParser`).

Reply:
so can we remove it?